### PR TITLE
Fixed remaining legacy dependencies

### PR DESCRIPTION
**Pull request by bdunogier**

### CODE REVIEW

**Comment by andrerom** [comment]
+1

File: Resources/views/line/blog_post.html.twig [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {# In order to use blocks and override ez_render_field's template within this file,
  # we need to extend from another template and surround our content with a block.
  #}
-{% extends viewbaseLayout %}
+{% extends 'EzPublishCoreBundle::viewbase_layout.html.twig' %}
```
Comment (crevillo) [inline Resources/views/line/blog_post.html.twig]: Ok, but why this is related to decoupling legacy?

File: composer.json [hunk 2]
```diff
@@ -15,7 +15,8 @@
         "ezsystems/comments-bundle": "@dev",
         "ezsystems/platform-ui-assets-bundle": "~0.1",
         "whiteoctober/breadcrumbs-bundle": "~1.0.1",
-        "knplabs/knp-menu-bundle": "1.*"
+        "knplabs/knp-menu-bundle": "1.*",
+        "zetacomponents/feed": "~1.4"
```
Comment (crevillo) [inline composer.json]: Hmmm, why? Do we need or will need it?

**Comment by crevillo** [comment]
My comments are just curiosity, +1 anyway.

File: composer.json (hunk 2 above)
Comment (lolautruche) [inline composer.json]: Yes it's needed for the feed block on the home page.

**Comment by lolautruche** [comment]
+1

File: Resources/views/line/blog_post.html.twig (hunk 1 above)
Comment (andrerom) [inline Resources/views/line/blog_post.html.twig]: @lolautruche this (afaik) is the workaround talked about in todays daily.

Comment (bdunogier) [inline Resources/views/line/blog_post.html.twig]: It is. And I can confirm that it still occurs if I revert this patch.

Trying to investigate.

Comment (andrerom) [inline Resources/views/line/blog_post.html.twig]: @dpobel is this the one you mean? `{% extends noLayout ? viewbaseLayout : "eZDemoBundle::pagelayout.html.twig" %}`

**Pull request merged**